fe/fe-core/src/main/java/org/apache/doris/catalog/OlapTable.java

```diff
@@ -1278,6 +1277,9 @@ public void readFields(DataInput in) throws IOException {
         if (in.readBoolean()) {
             tableProperty = TableProperty.read(in);
         }
+        if (isAutoBucket()) {
+            defaultDistributionInfo.markAutoBucket();
```

Comment:
You have read the `defaultDistributionInfo` in line 1243, so no need to mark auto bucket again.

Reply:
For compatibility with DistributuinInfo persistence， I store autoBucket in property.